compass/ocean/tests/global_ocean/files_for_e3sm/diagnostic_masks.py

```diff
@@ -82,8 +82,8 @@ def make_diagnostics_files(logger, mesh_short_name, with_ice_shelf_cavities,
 
     gf = GeometricFeatures()
     region_groups = ['Antarctic Regions', 'Arctic Ocean Regions',
-                     'Arctic Sea Ice Regions', 'Ocean Basins',
-                     'Ocean Subbasins', 'ISMIP6 Regions']
+                     'Arctic Sea Ice Regions', 'Greenland Regions',
+                     'Ocean Basins', 'Ocean Subbasins', 'ISMIP6 Regions']
```

Comment:
@trhille, let me know if you're okay with including this.  I'll run a quick test.

Reply:
I tested this with the `qu240_for_e3sm` and it produced the Greenland region masks as expected.

Reply:
Yeah, that's fine with me.